dubbo-config/dubbo-config-api/src/main/java/org/apache/dubbo/config/ServiceConfig.java

```diff
@@ -217,6 +219,33 @@ public void run() {
         }
     }
 
+    /**
+     * Tries to get the value(ms) of the global service delay from the VM options or the properties file.
+     *
+     * @return the value(ms) of the global service delay, <code>null</code> if it not be assigned.
+     */
+    private Integer getGlobalServiceDelay() {
+        // try to get DUBBO_SERVICE_DELAY from the VM options first
+        String delayProperty = System.getProperty(Constants.DUBBO_SERVICE_DELAY_KEY);
+
+        if (delayProperty == null) {

```

Comment:
Maybe use StringUtils is better.
Empty string (like "") or null seems like property delay didn't config.